app/models/user.rb

```diff
@@ -87,6 +87,12 @@ def watching?(task)
     self.tasks.where(id: task.id).exists?
   end
 
+  def search_tasks(search)
+    self.projects.each do |project|
```

Comment:
this code sucks

rework and use associations and READ ABOUT EACH / RETURN / MAP

it's strange to me this code

You NEED TO SEACH IN SCOPE OF ALL TASKS directly without loop by projects